src/software/SfM/main_openMVG2openMVS.cpp

```diff
@@ -107,11 +107,20 @@ bool exportToOpenMVS(
         ReadImage(srcImage.c_str(), &imageRGB);
         UndistortImage(imageRGB, cam, imageRGB_ud, BLACK);
         WriteImage(image.name.c_str(), imageRGB_ud);
+        // undistort mask and save it
+        auto maskFileName = srcImage + ".mask.png";
+        if (stlplus::is_file(maskFileName)) {
+          Image<openMVG::image::RGBColor> maskRGB, maskRGB_ud;
+          ReadImage(maskFileName.c_str(), &maskRGB);
```

Comment:
Can we add the same logic as here: https://github.com/openMVG/openMVG/blob/develop/src/software/SfM/main_openMVG2openMVS.cpp#L162

Sorry I just pushed this safety check some days ago.